arbnode/node.go

```diff
@@ -649,13 +653,14 @@ func createNodeImpl(
 		if !(config.SeqCoordinator.Enable || config.Sequencer.Dangerous.NoCoordinator) {
 			return nil, errors.New("sequencer must be enabled with coordinator, unless dangerous.no-coordinator set")
 		}
+		sequencerConfigFetcher := func() *SequencerConfig { return &configFetcher().Sequencer }
```

Comment:
Because this returns a pointer, it shouldn't copy the config, which is perfect :)